src/pages/blog/mage-os-update-2022-week-30.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: '@/layouts/post.astro'
+date: 2022-07-29
+title: Week 26-30 of 2022
```

Comment:
```suggestion
title: New Website, Mirror Distribution, Nightly Builds, Content & Community Updates
```

@rhoerr suggested to use more meaningful titles, which makes sense IMHO. I kind of dislike just listing the topics, but this is still better than a generic title.